src/modules/add-player.js

```diff
@@ -0,0 +1,42 @@
+import display from './display.js';
+
+const KEY = 'ogEBHq5P32FHEDzPh5SP';
+const BASE_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+const URL = `${BASE_URL}/games/${KEY}/scores/`;
+
+const addPlayerToAPI = (player) => {
+  fetch(URL, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify({
+      user: player.user,
+      score: player.score,
+    }),
+  })
+    .then((response) => {
+      const players = response.json();
+      display();
+      return players;
+    })
+    .then(() => {
+      // console.log('Success:', player.result);
+    })
+    .catch(() => {
+      // console.error('Error:', player);
+    });
```

Comment:
[OPTIONAL] Kindly, I suggest you remove the functions you are not using, that will make your code cleaner and more professional.

Reply:
Thanks, @mgmediaweb for the feedback, it's really helpful.